hack/generator/pkg/astmodel/identifier_factory.go

```diff
@@ -33,7 +33,8 @@ type IdentifierFactory interface {
 
 // identifierFactory is an implementation of the IdentifierFactory interface
 type identifierFactory struct {
-	renames map[string]string
+	renames       map[string]string
```

Comment:
I'm open to just using renames for this but that's case insensitive right now and in reality we only need these for the lowercase flavors.

This comes up when generating methods where I do something like:

```
func (c *TheType) ConvertToArm() *TheTypeArm {
    package := c.Package.ConvertToArm()
    ...
}
```

Reply:
Given that there's a different context/timing, having a separate map makes sense.